roles/rhods_test_jupyterlab/files/entrypoint.sh

```diff
@@ -35,10 +37,17 @@ mkdir -p "${ARTIFACTS_DIR}"
 
 trap "touch $ARTIFACTS_DIR/test.exit_code" EXIT
 
-sed "s/#{JOB_COMPLETION_INDEX}/${JOB_COMPLETION_INDEX:-1}/g" /mnt/ods-ci-test-variables/test-variables.yml > /tmp/test-variables.yml
+sed "s/#{JOB_COMPLETION_INDEX}/${JOB_COMPLETION_INDEX}/g" /mnt/ods-ci-test-variables/test-variables.yml > /tmp/test-variables.yml
 
 cp "/mnt/rhods-jupyterlab-entrypoint/$RUN_ROBOT_TEST_CASE" .
 
+# Sleep for a while to avoid DDoSing OAuth
+
+sleep_delay=$(python3 -c "print($JOB_COMPLETION_INDEX * $SLEEP_FACTOR)")
+
+echo "Waiting $sleep_delay seconds before starting (job index: $JOB_COMPLETION_INDEX, sleep factor: $SLEEP_FACTOR)"
+sleep "$sleep_delay"
```

Comment:
can you add this line?
```
echo "$sleep_delay" > "${ARTIFACTS_DIR}/sleep_delay"
```

Reply:
hum, I don't understand why this file created above
```
echo "$tries"> "$ARTIFACTS_DIR/oc_login.tries"
```
isn't saved 🤔 
Should be [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/417/pull-ci-openshift-psap-ci-artifacts-master-ods-jh-on-ocp/1544779271203983360/artifacts/jh-on-ocp/test/artifacts/008__driver_rhods__test_jupyterlab/ods-ci/ods-ci-0/) for instance

but this one created below is there as expected:
```
echo "$test_exit_code" > "${ARTIFACTS_DIR}/test.exit_code"
```

Reply:
> can you add this line?
> 
> ```
> echo "$sleep_delay" > "${ARTIFACTS_DIR}/sleep_delay"
> ```

Done, thanks